### ISSUE TITLE
Configure extra files to check

### ISSUE THREAD

**Issue by julienfalque** [maglnet/ComposerRequireChecker#76]
In the `bin/console` file of a Symfony project, a used symbol that is part of `require-dev` in the `composer.json` file is not reported by `composer-require-checker`. If I understand correctly, this is because `bin/console` is not part of an `autoload` directory (`src/`) and thus not checked at all. But the console will still crash after running `composer install --no-dev`.

Is there a way to make the tool check the file anyway?

**Comment by Ocramius** [comment]
Hi @julienfalque,

If a dependency is not in `"require"` then it is to be ignored by the tool, because consumers of your package may not get that dependency at all in first place (which is what this package is checking in first place), and that will lead to a crash.

If this is an exception to the rule, you'll need to configure it as such.

**Comment by julienfalque** [comment]
Sorry, I don't understand your answer. Isn't the purpose of the tool to report used symbols that are not part of `require`? That means those are *not* ignored, right?

In my case, if I move the file into the `src/` directory (and rename it to `console.php`), the tool does report the unknown symbol.

**Comment by Ocramius** [comment]
Oh, then I totally misunderstood. Re-opening
